api/models/Post.js

```diff
@@ -67,7 +67,10 @@ module.exports = bookshelf.Model.extend(Object.assign({
   },
 
   comments: function () {
-    return this.hasMany(Comment, 'post_id')
+    return this.hasMany(Comment, 'post_id').query({ where: {
+      'comments.active': true,
+      'comments.comment_id': null
```

Comment:
this seems fine for now but abstractly it makes sense that the comments() relationship should return all comments on the post to me, and maybe a parameter would say to only return top level comments? dont need to change this now just wanted to flag that